standalone-metastore/src/main/sql/oracle/hive-schema-2.3.0.oracle.sql

```diff
@@ -576,71 +574,7 @@ CREATE TABLE NOTIFICATION_SEQUENCE
 
 ALTER TABLE NOTIFICATION_SEQUENCE ADD CONSTRAINT NOTIFICATION_SEQUENCE_PK PRIMARY KEY (NNI_ID);
 
--- Tables to manage resource plans.
 
-CREATE TABLE WM_RESOURCEPLAN
```

Comment:
any particular reason why we are removing this stuff?

Reply:
Same comment as previous.